rmgpy/reaction.py

```diff
@@ -560,22 +560,37 @@ def get_equilibrium_constant(self, T, type='Kc'):
         Return the equilibrium constant for the reaction at the specified
         temperature `T` in K. The `type` parameter lets	you specify the
         quantities used in the equilibrium constant: ``Ka`` for	activities,
-        ``Kc`` for concentrations (default), or ``Kp`` for pressures. Note that
-        this function currently assumes an ideal gas mixture.
+        ``Kc`` for concentrations (default), or ``Kp`` for pressures.  This
+        function assumes a reference pressure of 1e5 Pa for gas phases species
+        and uses the ideal gas law to determine reference concentrations. For 
+        solid surface site species, a reference concentration of 1 is assumed. 
```

Comment:
added the reference from Cantera's source code https://github.com/Cantera/cantera/blob/master/include/cantera/kinetics/InterfaceKinetics.h#L467